pkg/providers/instance/instance.go

```diff
@@ -55,9 +53,13 @@ import (
 	"github.com/Azure/azure-sdk-for-go/sdk/resourcemanager/network/armnetwork"
 )
 
+var (
+	vmListQuery  string
+	nicListQuery string
+)
```

Comment:
This is probably a leftover from VM GC implementation, but not clear to me why these need to be global rather than DefaultProvider fields